app/controllers/api/containers_controller.rb

```diff
@@ -5,24 +5,32 @@ def index
 
   def create
     container = Container.new(container_params)
-    container.image = Image.find_by_image_id(params[:image_id])
-    container.save
-    render json: container
+    container.image = Image.find(params[:container][:image_id])
```

Comment:
Either you use *Image.find_by_image_id* and you handle the fact that it can be nil or you use *Image.find* but you need to throw handle all the "ActiveRecord::RecordNotFound" error that will probably happen.